package io.siggi.databackup.data;

public class DirectoryEntryException extends RuntimeException {
    public DirectoryEntryException(String message, Throwable cause) {
        super(message, cause);
    }
}
